wiring/src/string_convert.cpp

```diff
@@ -109,6 +109,55 @@ __attribute__((weak)) char* utoa(unsigned a, char* buffer, int radix) {
     return ultoa(a, buffer, radix);
 }
 
+__attribute__((weak)) char* ulltoa(uint64_t value, char* buffer, int radix, char pad) {
```

Comment:
`std::to_chars` should work for both all (including 64-bit) types, we should probably get rid of the current Arduino implementations and just streamline `std::to_chars` usage.

Reply:
I can go along with that

Reply:
done